Task Liverpool/src/main/java/pages/HomePage.java

```diff
@@ -0,0 +1,27 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+public class HomePage extends BasePage {
+
+    private By searchBar =By.id("mainSearchbar");
+    private By searchBarButton = By.cssSelector(".input-group-text .icon-zoom");
```

Comment:
do we need the `.icon-zoom` ?

Reply:
not really, I removed it. How did you know?